packages/loopring_v3/contracts/impl/LoopringV3.sol

```diff
@@ -139,16 +139,17 @@ contract LoopringV3 is ILoopringV3, Claimable
         emit ProtocolFeeVaultUpdated(protocolFeeVault);
     }
 
-    function createExchange(
+    function registerExchange(
+        address exchangeAddress,
+        address _owner,
         address payable _operator,
-        bool onchainDataAvailability
+        bool    onchainDataAvailability
```

Comment:
I should probably add noReentrence to this method.

Reply:
I don't think it's needed now, but because none of the functions are called a lot, so the gas cost doesn't matter, it's probably a good idea to just mark all external functions as `nonReentrant` for safety.

Reply:
Ok, will do

Reply:
I will do it in a separate PR.